libafl/Cargo.toml

```diff
@@ -58,6 +58,8 @@ num_enum = { version = "0.5.1", default-features = false }
 typed-builder = "0.9.0" # Implement the builder pattern at compiletime
 ahash = { version = "0.7", default-features=false, features=["compile-time-rng"] } # The hash function already used in hashbrown
 intervaltree = { git = "https://github.com/andreafioraldi/rust-intervaltree", version = "0.2.6", default-features = false, features = ["serde"] }
+digest = "0.9.0"
```

Comment:
This appears to be unused

Reply:
Oh yeah I was using it to compute the hash. Will remove